stylesheets/components/avatar.css

```diff
@@ -0,0 +1,35 @@
+.avatar {
+  border-radius: 50%;
+  width: 3.5rem;
+  height: 3.5rem;
+  margin: 0 0.5rem;
+}
+
+.xs {
```

Comment:
Instead of using xs, use the class name as avatar-xs. 
It might cause conflicts in the future.
(Apply this in all)

Reply:
Thanks, that helps! Made the changes